rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -77,6 +78,41 @@ public class BridgeUtils {
 
     private BridgeUtils() {}
 
+    /*
+    Received tx inputs are replaced by base inputs without signatures that spend from the given federation.
+    This way the tx has the same hash as the one registered in release_requested event topics.
+     */
+    public static void removeSignaturesFromTransaction(BtcTransaction tx, Federation spendingFed) {
+        for (int inputIndex = 0; inputIndex < tx.getInputs().size(); inputIndex++) {
+            //Get redeem script for current input
+            TransactionInput txInput = tx.getInput(inputIndex);
+            Script inputRedeemScript = getRedeemScriptFromInput(txInput);
+            logger.trace("[removeSignaturesFromTransaction] input {} scriptSig {}", inputIndex, tx.getInput(inputIndex).getScriptSig());
```

Comment:
No big deal, but I think it's not necessary to use the `tx.getInput(inputIndex).getScriptSig()` line here, because the object that `tx.getInput(inputIndex)` returns is already assigned above in `txInput`. So maybe `txInput..getScriptSig()` will be enough in this line.